File: Server/src/main/java/com/github/dr/rwserver/util/io/Streams.java
package com.github.dr.rwserver.util.io;

import java.io.Closeable;

public class Streams {
    /** Close and ignore all errors. */
    public static void close(Closeable c){
        if(c != null){
            try{
                c.close();
            }catch(Throwable ignored){
            }
        }
    }
}

File: Server/src/main/java/cn/rwhps/server/util/io/Streams.kt
/*
 * Copyright 2020-2022 RW-HPS Team and contributors.
 *
 * 此源代码的使用受 GNU AFFERO GENERAL PUBLIC LICENSE version 3 许可证的约束, 可以在以下链接找到该许可证.
 * Use of this source code is governed by the GNU AGPLv3 license that can be found through the following link.
 *
 * https://github.com/RW-HPS/RW-HPS/blob/master/LICENSE
 */

package cn.rwhps.server.util.io

import java.io.Closeable

object Streams {
    /** Close and ignore all errors.  */
    fun close(c: Closeable?) {
        if (c != null) {
            try {
                c.close()
            } catch (ignored: Throwable) {
            }
        }
    }
}